recipe/meta.yaml

```
@@ -1,27 +1,28 @@
 {% set name = "langchain-text-splitters" %}
-{% set version = "0.3.3" %}
+{% set version = "0.3.8" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-','_') }}-{{ version }}.tar.gz
-  sha256: c596958dcab15fdfe0627fd36ce9d588d0a7e35593af70cd10d0a4a06d69b3ee
+  sha256: 116d4b9f2a22dda357d0b79e30acf005c5518177971c66a9f1ab0edfdb0f912e
 
 build:
-  skip: true  # [py<39 or s390x]
+  skip: true  # [py<39]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
     - python
-    - poetry-core >=1.0.0
+    - pdm-backend
     - pip
   run:
     - python
-    - langchain-core >=0.3.15,<0.4.0
+    - langchain-core
```

```suggestion
    - langchain-core <1.0.0,>=0.3.51
```
see https://github.com/langchain-ai/langchain/blob/langchain-text-splitters%3D%3D0.3.8/libs/text-splitters/pyproject.toml#L9